infra/mock_api/template/ecs.tf

```diff
@@ -109,21 +106,132 @@ resource "aws_ecs_task_definition" "mock-api-ecs-task-definition" {
         {
           containerPort : 8080
         }
-      ],
+      ]
+      readonlyRootFilesystem = true
+      linuxParameters = {
+        capabilities = {
+          drop = ["ALL"]
+        },
+        initProcessEnabled = true
+      }
+      environment : [
```

Comment:
Could we also set: `LOG_FORMAT` as `json` so anything running in AWS auto-uses JSON logs and not the human-readable format. Happy to follow-up and add this.